docs/wiki/end-user-guide/preparing-your-modpack.md

```diff
@@ -0,0 +1,94 @@
+| **Raw Input**                   | Already included in Cleanroom.                              | :no_entry_sign: Remove                                                                                                                                                                                     |
+| **SerializationIsBad**          | Redundant. Modern Java versions fix the issue this solves   | :no_entry_sign: Remove                                                                                                                                                                                     |
+| **TexFix**                      | Merged and rewritten in CensoredASM                         | :white_check_mark: Use **[CensoredASM](https://www.curseforge.com/minecraft/mc-mods/lolasm)**                                                                                                              |
+| **VanillaFix**                  | Merged and improved in CensoredASM                          | :white_check_mark: Use **[CensoredASM](https://www.curseforge.com/minecraft/mc-mods/lolasm)**                                                                                                              |
+| **Secret Rooms**                | Has an issue with updater                                   | :no_entry_sign: Disable the update checker in config file                                                                                                                                                  |
+
+::: info Note {id="note"}
+If your modpack uses **Bansoukou**, make sure to **run the instance at least once** before installing cleanroom loader so its patches are applied.
+:::
+## Performance / Memory optimisation mods
+The most important optimisation mods for **Minecraft 1.12.2** can be summed up to the following list:
+- **[CensoredASM](https://www.curseforge.com/minecraft/mc-mods/lolasm)** : Huge optimizations that are targeted at Minecraft 1.12.2
+- **[VintageFix](https://www.curseforge.com/minecraft/mc-mods/vintagefix)** : Load time and RAM use improvements for Minecraft 1.12.2
+- **[Universal Tweaks](https://www.curseforge.com/minecraft/mc-mods/universal-tweaks)** : A community project to consolidate various bugfixes and tweaks into a single solution for Minecraft 1.12.2
+- **[Stellar Core](https://www.curseforge.com/minecraft/mc-mods/stellarcore)** : Provides extensive bug fixes, performance improvements, and additional features across a wide range of mods
+::: info Note {id="note"}
+Universal Tweaks Supercedes a lot of mods which will not be mentionned here. Please consult the list by clicking this [**Link**](https://legacy.curseforge.com/minecraft/mc-mods/universal-tweaks/files/6953147)
```

Comment:
maybe note that "this can be observed via an in-game warning or by looking at the incompatible mods list [here]"
also, `Supercedes` -> `supersedes`